### DISCUSSION TITLE
Issues with Sentry

### DISCUSSION THREAD

**Discussion by olaj** [vercel/next.js#12275] [Help]
I have messed around with Next and Sentry the last couple of days and i have some questions.

I have cloned the "with-sentry-simple" example and have configured sentry and so on. My main issue is with the sitemaps. I'm running the project with yarn build / yarn start locally. I have not deployed it to a server.

1. When i generate a server side error it gets logged to sentry and I get this: https://i.imgur.com/d6jXnv0.png, should it look like that with a sourcemap? Do i have to too high exceptions and expected it to show the exact JS file as it look in the project? Or is something not working correctly? 

2. The clientside errors doesn't show up at all in sentry.

Am i'm doing something wrong and in that case what? Let me know if you want more info.

**Comment by Vadorequest** [comment]
@olaj You may be interested by https://github.com/UnlyEd/next-right-now/issues/28

Branch (preset) where Sentry is configured:
https://github.com/UnlyEd/next-right-now/tree/v1-ssr-mst-aptd-gcms-lcz-sty

**Reply by olaj** [reply]
Interesting, will look into that more tomorrow. On my mobile now. But as far as I understand it the issue is not really solved and what I get there is expected behavior? As you say it’s maybe acceptable but a bit irritating and weird that it isn’t mentioned more? 

I still don’t understand why the ‘client side’ errors is not sent to Sentry either.

**Reply by olaj** [reply]
Will grab a lot from that project in any case, looks good!

**Reply by Vadorequest** [reply]
If you read down all the comments, (I guess you did) you'll see that we consider it "okay". It's hard to tell "how good" error reporting can become with source maps, and it seems to be affected by the "runtime engine" (server vs browser)

But our setup does work with both server and browser, it's just the quality of the error reporting that differs a bit, but it's still acceptable (to my taste).

Take a look at the videos and you'll see how good/bad it is, for each runtime.

**Comment by olaj** [comment]
So i found out why the client errors wasn't logged.

I had the "Filter event from localhost" turned on in Sentry

¯\_(ツ)_/¯

**Reply by Vadorequest** [reply]
Always test on a production-ish environment 😉
